packages/terra-site/webpack.config.js

```diff
@@ -109,4 +109,7 @@ module.exports = {
       errors: true,
     },
   },
+  resolveLoader: {
```

Comment:
How does this differ from
```js
resolve: {
  modules: [path.resolve(path.join(__dirname, 'node_modules'))],
}
```
We use something like that for translations:

https://github.com/cerner/terra-core/blob/4a8c7bff4223a28aa016686a82e1fb7fe34d9b49/packages/terra-site/webpack.config.js#L76-L78

Can you move `resolveLoaders` up so it follows the `resolve` key.